cluster/gce/util.sh

```diff
@@ -1373,7 +1385,7 @@ function kube-down() {
 function get-replica-name() {
   echo $(gcloud compute instances list \
     --project "${PROJECT}" \
-    --zone "${ZONE}" \
+    --zones "${ZONE}" \
```

Comment:
Assume the flag is renamed? I saw other scripts still using --zone and might need to be renamed